models.cpp

```diff
@@ -634,3 +635,30 @@ std::valarray<double> Models::integrate_ode_explicit_euler(const Models::explEul
 
   return x;
 }
+
+// ODE solver for saving many times
+std::vector<std::valarray<double>> Models::integrate_ode_ee_many_times(const std::valarray<double>& init_condition,
```

Comment:
You could have just called this function `integrate_ode_explicit_euler()` as well. It has a different argument list from the other function of same name to distinguish it from the other one, so there is no conflict of names.

Reply:
I see. I'm still not used to thinking of operator overloading when I program. I want to go through my naming conventions anyway while the project is still small because I know there are some inconsistencies, so I'll keep this in mind for when I do that.

Reply:
You aren't overloading operators, just regular functions.